PendingReleaseNotes

```diff
@@ -132,3 +132,7 @@ of the feature, refer this link on how to perform it:
 https://docs.ceph.com/en/quincy/cephadm/upgrade/#staggered-upgrade
 Relevant tracker: https://tracker.ceph.com/issues/55715
 
+* New MDSMap field `reject_client_session` that can be set using the
+`fs set` command. This MDSMap field allows to block any incoming session
+request from client(s), this would be useful during some recovery situations
+where having MDS up might be useful but not clients.
```

Comment:
```suggestion
* Introduced a new file system flag `reject_client_session` that can be set using the
`fs set` command. This flag allows blocking any incoming session
request from client(s). This can be useful during some recovery situations
where it's desirable to bring MDS up but have no client workload.
```

Reply:
Done, Thanks!